sessionctx/variable/session.go

```diff
@@ -1423,6 +1423,9 @@ type SessionVars struct {
 	// If there exists an index whose estimated selectivity is smaller than this threshold, the optimizer won't
 	// use the ExpectedCnt to adjust the estimated row count for index scan.
 	OptOrderingIdxSelThresh float64
+
+	// OptimizerFixControl control some details of the optimizer behavior through the tidb_opt_fix_control variable.
+	OptimizerFixControl map[uint64]string
```

Comment:
How does this variable affect the optimizer? It doesn't seem to be reflected in this PR

Reply:
We only introduce this variable now, so we can rely on this in some future bug fixes or behavior changes that might cause regression.